### PR TITLE
Stratagem Updates

### PR DESCRIPTION
**Pull request by johnsonw**
- Update button text to read "Update"
- Update button text to read "Enable"
- Update button text to read "Delete"
- Show last known scan time on inode chart (Still need to do this for
    the file size distribution graph)
- Always display the inode table and the graph

Signed-off-by: Will Johnson <wjohnson@whamcloud.com>

### CODE REVIEW

**Comment by johnsonw** [comment]
Note that I still need to add the "last known scan" to the chart.

**Comment by johnsonw** [comment]
image

File: iml-wasm-components/iml-stratagem/src/inode_table.rs [hunk 1]
```diff
@@ -85,8 +89,19 @@ pub fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg>) {
...
                         .flatten()
-                        .map(|(_, uid, count)| INodeCount { uid, count })
+                        .map(|(timestamp, uid, count)| INodeCount {
+                            timestamp,
+                            uid,
+                            count,
+                        })
                         .collect();
+
+                    let ts = model.inodes.first().map(|x| x.timestamp.clone());
```
Comment (jgrund) [inline iml-wasm-components/iml-stratagem/src/inode_table.rs]:
```rust
model.last_known_scan = model.inodes.first().map(|x| get_date_time(x.timestamp))
```

Comment (johnsonw) [reply iml-wasm-components/iml-stratagem/src/inode_table.rs]: Nice, thanks.

File: iml-wasm-components/iml-stratagem/src/inode_table.rs [hunk 2]
```diff
@@ -1,5 +1,7 @@
 use crate::inode_error::InodeError;
 use bootstrap_components::bs_table;
+use chrono::offset::TimeZone;
+use chrono::offset::Utc;
```
Comment (jgrund) [inline iml-wasm-components/iml-stratagem/src/inode_table.rs]: Combine these into a single import.

**Pull request merged**